pkg/rancher-desktop/utils/_demo_marketplace_items.js

```diff
@@ -1,5 +1,31 @@
 export const demoMarketplace = {
   summaries: [
+    {
+      id:                    '',
+      name:                  'Open WebUI Extension',
+      containerd_compatible: true,
+      slug:                  'ghcr.io/rancher-sandbox/rancher-desktop-rdx-open-webui',
+      type:                  'extension',
+      publisher:             { name: 'SUSE LLC' },
+      created_at:            '2024-10-03T15:43:34Z',
+      updated_at:            '2024-10-03T15:43:34.144218728Z',
+      short_description:     'Open WebUI and Ollama packaged into a docker extension for local GenAI development',
+      source:                'community',
+      extension_reviewed:    true,
```

Comment:
It doesn't really matter because we don't use this data anywhere, but this setting means that the extension has been reviewed by Docker Inc, which is not the case.
```suggestion
      extension_reviewed:    false,
```